### ISSUE TITLE
Forge - Invalid module name: '' is not a Java identifier (Minecraft 1.20.4 and up)

### ISSUE THREAD

**Issue by Rex109** [architectury/architectury-loom#215]
Minecraft just hangs on the start screen until i force quit it. The project is freshly generated from https://generate.architectury.dev/ and does not contain any type of additional code.

It spits this error:
Caused by: java.lang.IllegalArgumentException: : Invalid module name: '' is not a Java identifier

Complete trace:
https://gnomebot.dev/paste/1243356946950524949/1243371164009627759/1243371163585875988

**Comment by ShayBox** [comment]
Same issue

**Comment by boholder** [comment]
Same after upgrading to 1.20.6 and config `jopt-simple` and comment out `remapJar` task in forge's `gradle.build` according to https://discord.com/channels/792699517631594506/1242850221025656932

env:
minecraft_version= 1.20.6
forge_version= 50.0.31
architectury-plugin = 3.4-SNAPSHOT
dev.architectury.loom  = 1.6-SNAPSHOT

**Comment by Jab125** [comment]
Add this to forge's `build.gradle` 
```groovy
// Forge gets the module name from `Automatic-Module-Name`, so this is required to properly load on Forge.
processResources {
  doLast {
    java.nio.file.Path.of(outputs.files.asPath).resolve("META-INF/MANIFEST.MF").text = "Manifest-Version: 1.0\nAutomatic-Module-Name: ignoreme\n"
  }
}
```
Note that you still can't launch MC until #216 is fixed (#219)
